crates/turborepo-filewatch/src/package_watcher.rs

```diff
@@ -181,6 +181,7 @@ const INVALIDATION_PATHS: &[&str] = &[
     "package-lock.json",
     "yarn.lock",
     "bun.lockb",
+    // "bun.lock", // Not yet.
```

Comment:
happy to remove this if it's not helpful, but thought it might be

Reply:
I think adding this would make sense, https://github.com/vercel/turborepo/pull/9783 is out in the wild

Reply:
great: I put up https://github.com/vercel/turborepo/pull/9958 and removed it from this PR in 1293f7765e438bd3737636fda606882879b117c4